### PR TITLE
Customer service and its implementation class

### PR DESCRIPTION
**Pull request by sagarxt**
#17

### CODE REVIEW

File: backend/src/main/java/com/moshop/backend/services/impl/CustomerServiceImpl.java
Comment (Abinashkjcjanscinanconanc) [inline backend/src/main/java/com/moshop/backend/services/impl/CustomerServiceImpl.java:1]: add sapce in line no. 35, 36, 46 and 50

**Review by Abinashkjcjanscinanconanc — COMMENTED** [review]
Change accordingto comments

File: backend/src/main/java/com/moshop/backend/services/impl/CustomerServiceImpl.java
Comment (sagarxt) [reply backend/src/main/java/com/moshop/backend/services/impl/CustomerServiceImpl.java:1]: space added in add line no. 35, 36, 46 and 50

**Review by Abinashkjcjanscinanconanc — APPROVED** [review]
LGTM

**Pull request merged**